pubsub/kafka/kafka.go

```diff
@@ -47,7 +47,8 @@ type Kafka struct {
 
 type kafkaMetadata struct {
 	Brokers         []string `json:"brokers"`
-	ConsumerID      string   `json:"consumerID"`
+	ConsumerGroup   string   `json:"consumerGroup"`
```

Comment:
Please, keep support for consumerID too to avoid breaking people that are using this. This component is not GA but Kafka is somewhat popular and backwards compatibility here is a small cost.